ncm-grub/src/test/resources/nopassword.pan

```diff
@@ -0,0 +1,13 @@
+object template nopassword;
+
+include 'base';
+
+
+prefix "/software/components/grub/password";
+"enabled" = false;
+"password" = "";
```

Comment:
is this intentionally empty? i wonder why we allow an empty string as a password

Reply:
Mostly accidental? Just an artifact of testing. I was trying with `enabled=true && password=""` which I believe was accepted and gave an empty `password=` on the command line.

Reply:
but i guess empty `password=` makes no sense?
can you tighten the schema https://github.com/quattor/configuration-modules-core/blob/master/ncm-grub/src/main/pan/components/grub/schema.pan#L25, use `string(1..)` as type